authorizer-app-backend/src/main/java/org/radarbase/authorizer/service/OAuth1RestSourceAuthorizationService.kt

```diff
@@ -146,6 +146,15 @@ abstract class OAuth1RestSourceAuthorizationService(
             .build()
     }
 
+    override fun signUrl(user: RestSourceUser, url: String, method: String, params: MutableMap<String, String?>): String {
+        val authConfig = configMap[user.sourceType]
+            ?: throw HttpBadRequestException("client-config-not-found",
+                "Cannot find client configurations for source-type ${user.sourceType}")
+        val tokens = RestOauth1AccessToken(user.accessToken!!, user.refreshToken)
```

Comment:
Instead of asserting an access token exists with `!!`, it would be better to check that it exists or throw a proper HTTP exception.